dataland-backend/src/main/kotlin/org/dataland/datalandbackend/DatalandBackend.kt

```diff
@@ -29,5 +29,5 @@ class DatalandBackend : OpenAPIConfiguration
  * Main function to be executed for running the spring boot dataland backend process
  */
 fun main(args: Array<String>) {
-    runApplication<DatalandBackend>(*args)
+    runApplication<DatalandBackend>(args = args)
```

Comment:
why this change?

Reply:
I see - to be consistent with the other spring boot services. I like that.

Reply:
Also detekt (I think) said that it would be slower the other way because it would walk through the whole list and not just pass the references.